### ISSUE TITLE
[Bug] Bug description - Development hint installation is giving error

### ISSUE THREAD

**Issue by NehaOberoi92** [webhintio/hint#1852]
Hi Team, I am facing issue while using below approach for development hints installation.
 {
    "extends": ["development"]
}

Error : { ResourceError: Module typescript not found when loading @hint/hint-typescript-config
    at Object.exports.tryToLoadFrom (/usr/local/lib/node_modules/hint/dist/src/lib/utils/resource-loader.js:140:19)
    at getResource (/usr/local/lib/node_modules/hint/dist/src/lib/utils/resource-loader.js:147:30)
    at sources.some (/usr/local/lib/node_modules/hint/dist/src/lib/utils/resource-loader.js:209:21)
    at Array.some (<anonymous>)
    at Object.exports.loadResource (/usr/local/lib/node_modules/hint/dist/src/lib/utils/resource-loader.js:208:13)
    at items.reduce 

Can you guide me how to use development hints ? Also I would like to know are web recommended hints enough or what are the hints which you recommend among the - https://webhint.io/docs/user-guide/hints/

**Comment by antross** [comment]
@NehaOberoi92 Thanks for the report! This is a bug on our end, but if you want to workaround it while we get the issue fixed you can run `npm install -g typescript`, then try again.

@webhintio/core Looks like `@hint/hint-typescript-config` currently lists `typescript` as a `"devDependency"`, but actually has a runtime depedency due to a reference to the `ScriptTarget` enum (introduced in PR #1578 a couple of months ago). Everything else that's referenced is just compile-time types. We probably didn't notice because most of us already have `typescript` installed globally. While we could move `typescript` to be a `"dependency"`, that's brings in a lot more than we need at runtime. After talking with @sarvaje I'm pretty sure we don't need this enum reference as we're maintaining an internal map anyway.

TL;DR I recommend we fix the issue by removing the use of `ScriptTarget` (keeping `typescript` as a `"devDependency"`). I'll open a PR shortly.
